src/ui/public/saved_objects/angular_service.js

```diff
@@ -0,0 +1,7 @@
+import { uiModules } from 'ui/modules';
+import chrome from 'ui/chrome';
+import { SavedObjectsClient } from 'ui/saved_objects';
+
+uiModules.get('kibana').service('savedObjectsClient', function ($http, $q) {
```

Comment:
Rather than relying on angular DI and `ui/autoload/modules`, can we just export a `SavedObjectsClientProvider` here and use `Private(SavedObjectsClientProvider)` to get the client?